_posts/2018-07-12-OpenRefine-Fellow.md

```diff
@@ -0,0 +1,26 @@
+---
+  layout: post
+  author: magdmartin
+  title: OpenRefine Fellow
+  category: My Category
+---
+
+The OpenRefine Foundation is looking to hire a full-time Fellow for four months period to help us have a constant, guarantee attention to address our roadmap instead of volunteer-only efforts. The position of the Fellow is primarily to help the community address major architecture changes including (see details in GitHub)
+* Replace ORG.JSON library - [#1652](https://github.com/OpenRefine/OpenRefine/issues/1652)
+* Improve Datagrid performance [#1347](https://github.com/OpenRefine/OpenRefine/issues/1347)
+* Research Apache Arrow to improve in-memory data model [#1469](https://github.com/OpenRefine/OpenRefine/issues/1469)
+* Prepare the front and back-end separation [#1483](https://github.com/OpenRefine/OpenRefine/issues/1483)
+
+Being a committer isn't a prerequisite for this position; we'll consider applications from anyone with a proven history of working with either the OpenRefine community or another similar open-source community.
+
+Your geographical location isn't important either - we have several methods of remote communication and coordination that we can use depending on the time zone difference to the supervising members of OpenRefine.
+
+You'll be expected to post a weekly report of your work to the [openrefine-developers mailing list](https://groups.google.com/forum/#!forum/openrefine-dev) and answer to the community feedback.  Since you will be working on core aspects of OpenRefine, we need to engage with all members of the community (commiters, contributors, extension maintainers) to gather feedbacks and manage the transition. 
+
+If you don't perform the duties to a satisfactory level, we may end your contract. Based on our funding the position is currently set for four months from September 1st to December 31 2018. Compensation isn't competitive with full-time salaries in big cities like San Francisco or London. The Fellow will be selected to make best use of available funds.
+
+If you're interested in applying for the position, please email us at: fellow@openrefine.org. Please add details of your experience with OpenRefine and open-source contribution and community support in general, the amount of time each week you'd like to dedicate to the position (a minimum of 30 hours a week), your hourly rate, and when you'd like to start working. 
+
+Applications will be open until August 3rd 2018 with the expectation that the successful candidate will be announced around August 20th, 2018. 
+
+Successful applicants will not be an employee of the OpenRefine Project or OpenRefine Foundation. Fellows will be contractors and expected to ensure that they meet all of their resident country's criteria for self-employment or having or being employed by a consulting company, invoicing the OpenRefine Foundation on a monthly basis and ensuring they pay all relevant taxes.
```

Comment:
Break up this sentence.
1. Fellow will be contractors and blah...
2. Fellows will be required to invoice the OpenRefine Foundation blah...